config.py

```diff
@@ -14,6 +14,8 @@
     config.get("database", "database_connection_string")
 )
 
+deactivate_scheduler = config.getboolean("scheduler", "deactivate", fallback=False)
```

Comment:
It would be nice to have such changes in an extra PR, makes it easier to review and reason about. Fine for now, just as a wish for the future.